CHANGELOG

```diff
@@ -1,3 +1,7 @@
+* Mon Jun 14 2021 Chris Tessmer <chris.tessmer@onyxpoint.com> - 6.6.0
```

Comment:
6.5.0 has not been released and the previous commit had removed Puppet 5 support.  I will fix the version and changelog appropriately.